src/platform/datapath_raw_xdp.c

```diff
@@ -832,3 +857,22 @@ CXPLAT_THREAD_CALLBACK(CxPlatXdpWorkerThread, Context)
 
     return 0;
 }
+
+CXPLAT_THREAD_CALLBACK(CxPlatXdpExtraWorkerThread, Context)
+{
+    XDP_DATAPATH* Xdp = (XDP_DATAPATH*)Context;
+
+    if (Xdp->Affinitize) {
+        GROUP_AFFINITY Affinity = {0};
+
+        Affinity.Group = Xdp->DatapathCpuGroup;
+        Affinity.Mask = (ULONG_PTR)1 << Xdp->DatapathCpuNumber;
+        SetThreadGroupAffinity(GetCurrentThread(), &Affinity, NULL);
+    }
+
+    while (Xdp->Running) {
+        CxPlatTimeUs64();
```

Comment:
What does this while loop do? Wouldn't this lock up the machine?

Reply:
Oh this is just a placeholder?

Reply:
This slipped into this PR accidentally. It is a thread meant to contend with the normal polling thread, to test the effect on latency. It's just meant to run at 100%.